### PR TITLE
Default to FOMO Nouns for Settlement 

### PR DESCRIPTION
**Pull request by bcjgit**
# TLDR

- Replaces `Settle Auction` button with `Help mint the next Noun` button
- This button links (in a new tab) to fomonouns.wtf
- Adds copy under `Help mint the next Noun` button to allow for force settlement (legacy settlement mechanic) 

# Screenshots 

## Mobile
Screen Shot 2022-01-06 at 1 52 39 PM

## Desktop
Screen Shot 2022-01-06 at 1 52 19 PM

# Testing Done
1. Confirmed that when `auctionEnded` is `true` then new experience is shown, otherwise not
2. Confirmed that clicking `Help mint the next Noun` button links (in a new tab) to fomonouns.wtf
3. Confirmed that clicking the emergency settlement link brings up the settlement modal (because there is no active auction right now, it correctly displays a modal telling me there is no active auction)

### CODE REVIEW

File: packages/nouns-webapp/src/components/Bid/index.tsx [hunk 1]
```diff
@@ -247,13 +255,28 @@ const Bid: React.FC<{
...
-        <Button
-          className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
-          onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
-          disabled={isDisabled}
-        >
-          {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
-        </Button>
+        {
+          !auctionEnded ? (
+              <Button
```
Comment (bcjgit) [inline packages/nouns-webapp/src/components/Bid/index.tsx:260]: I made the conscious choice to keep the active auction state exactly equal to status quo to make it easy to roll this back if we so choose. Happy to also remove the `auctionEnded` state code from there if people think that'd be better

File: packages/nouns-webapp/src/components/Bid/index.tsx [hunk 2]
```diff
@@ -247,13 +255,28 @@ const Bid: React.FC<{
...
+          ) 
+          : (
+            <>
+              <Button
+                className={classes.bidBtnAuctionEnded}
+                onClick={fomoNounsBtnOnClickHandler}
+              >
+                Help mint the next Noun ⌐◨-◨  
+            </Button>
+            <p className={classes.emergencySettleWrapper}>In case of emergency, <button onClick={settleAuctionHandler} className={classes.emergencySettleButton}>settle the auction manually.</button></p>
```
Comment (lastpunk9999) [inline packages/nouns-webapp/src/components/Bid/index.tsx:276]: When a wallet isn't connected, the original "Settle Auction" button was disabled. Should the emergency settle button follow the same logic?

**Comment by lastpunk9999** [comment]
I'm in favor of cdt.eth's comments copy changes (https://discord.com/channels/849745721544146955/855538581434007635/928744021168377926), but would add just a bit more disincentive to settling manually:
Button = `Vote for the next Noun ⌐◨-◨`
Emergency Settle = `Pay to settle manually`

An alternate option is to add some attention/color by using `_YesVoteIcon` to the left of the text instead of the glasses ASCII. Something like: `✔  Vote for the next Noun`

File: packages/nouns-webapp/src/components/Bid/index.tsx (hunk 1 above)
Comment (lastpunk9999) [reply packages/nouns-webapp/src/components/Bid/index.tsx:260]: This is a good move, as it allows anyone to click the FOMO button vs the Settle button logic which only allowed connected wallets to click.

**Comment by lastpunk9999** [comment]
I appreciate if this is out of scope, but as mentioned in Discord (https://discord.com/channels/849745721544146955/892962966226473100/928318258807971871), it would be great to have a way for visitors to nouns.wtf to know about fomo nouns ahead of the auction ending. My suggestion was to have a link always present below the bid area that would link out. Copy could be `Preview the next Noun` or `Help mint the next Noun`

**Comment by christiandavidturner** [comment]
> Emergency Settle = Pay to settle manually

Even better, @lastpunk9999.

**Comment by bcjgit** [comment]
> I appreciate if this is out of scope, but as mentioned in Discord (https://discord.com/channels/849745721544146955/892962966226473100/928318258807971871), it would be great to have a way for visitors to nouns.wtf to know about fomo nouns ahead of the auction ending. My suggestion was to have a link always present below the bid area that would link out. Copy could be `Preview the next Noun` or `Help mint the next Noun`

I think this makes a lot of sense. Happy to do this as a follow up PR today / tomorrow!

File: packages/nouns-webapp/src/components/Bid/index.tsx (hunk 2 above)
Comment (bcjgit) [reply packages/nouns-webapp/src/components/Bid/index.tsx:276]: +1 to this ... I'll add it ...

Comment (bcjgit) [reply packages/nouns-webapp/src/components/Bid/index.tsx]: Done. Made it so this copy only shows for connected wallets

**Comment by bcjgit** [comment]
@lastpunk9999 Updated copy and disabled state per your comments

### Disconnected wallet state
Screen Shot 2022-01-07 at 11 17 09 AM

### Connected wallet state 
Screen Shot 2022-01-07 at 11 17 03 AM

**Comment by bcjgit** [comment]
> I appreciate if this is out of scope, but as mentioned in Discord (https://discord.com/channels/849745721544146955/892962966226473100/928318258807971871), it would be great to have a way for visitors to nouns.wtf to know about fomo nouns ahead of the auction ending. My suggestion was to have a link always present below the bid area that would link out. Copy could be `Preview the next Noun` or `Help mint the next Noun`

https://github.com/nounsDAO/nouns-monorepo/pull/321

**Review by cryptoseneca — APPROVED** [review]

File: packages/nouns-webapp/src/components/Bid/index.tsx [hunk 3]
```diff
@@ -247,13 +257,35 @@ const Bid: React.FC<{
...
+          : (
+            <>
+              <Button
+                className={classes.bidBtnAuctionEnded}
+                onClick={fomoNounsBtnOnClickHandler}
+              >
+                Vote for the next Noun ⌐◨-◨
+            </Button>
+            {/* Only show force settle button if wallet connected */}
+            {isWalletConnected ? (<p className={classes.emergencySettleWrapper}>
```
[the remark this answers is not in the document]
Comment (bcjgit) [reply packages/nouns-webapp/src/components/Bid/index.tsx:279]: Got it. Will do. Wasn't sure if it was ok to use an a tag outside of the `Link` component. 

Def a low priority thing but it actually might be helpful from a dev velocity point of view to have a rough style guide.

[the remark this answers is not in the document]
Comment (bcjgit) [reply packages/nouns-webapp/src/components/Bid/index.tsx:279]: Oh wait ... I read this too quickly. 

The reason I opted for button here is that using an `a` tag requires an href, which we don't actually need in this case. The linter was complaining about having a `#` as the href.  Using a button styled as a link seemed to be the best solution based on what I read.

[the remark this answers is not in the document]
Comment (bcjgit) [reply packages/nouns-webapp/src/components/Bid/index.tsx]: Addressed the 

> Since the fallback is to do nothing, it might make sense to use && here instead of ? :. e.g.:

comment

[the remark this answers is not in the document]
Comment (cryptoseneca) [reply packages/nouns-webapp/src/components/Bid/index.tsx]: ah yes ` If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles.` missed that warning. all good!

[the remark this answers is not in the document]
Comment (lastpunk9999) [reply packages/nouns-webapp/src/components/Bid/index.tsx]: > Screen Shot 2022-01-07 at 11 17 09 AM

Since the button is black and text is white, it might look better to use the alternate ASCII: ⌐◧-◧

**Review by solimander — APPROVED** [review]
